src/rules/valid-author.ts

```diff
@@ -0,0 +1,48 @@
+import type { AST as JsonAST } from "jsonc-eslint-parser";
+
+import * as ESTree from "estree";
+import { validateAuthor } from "package-json-validator";
+
+import { createRule } from "../createRule.js";
+
+export const rule = createRule({
+	create(context) {
+		return {
+			"Program > JSONExpressionStatement > JSONObjectExpression > JSONProperty[key.value=author]"(
+				node: JsonAST.JSONProperty,
+			) {
+				const authorValue = JSON.parse(
+					context.sourceCode.getText(
+						node.value as unknown as ESTree.Node,
+					),
+				) as string;
```

Comment:
❔Is this certain to be a string?

Reply:
Setting `authorValue` to `unknown` should work
`const authorValue: unknown = JSON.parse(`